src/members/grid-list.jsx

```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { memberShape } from './model';
+import MembersGridListItem from './grid-list-item';
+
+export default function MembersGridList(props) {
+  const { members } = props;
```

Comment:
We moved all props destructuring info the function in other files.

So it would be `function MembersGridList({ members })`.

Reply:
fixed thanks :)